content/post/kubedb-v2024.08.16/index.md

```diff
@@ -0,0 +1,1689 @@
+| -                                              | `SetHostnameAsFQDN *bool` (new field)                       |
+| -                                              | `OS *core.PodOS` (new field)                                |
+| `continerSecurityContext`                      | `spec.containers[i].containerSecurityContext`               |
+| `resources`                                    | `spec.containers[i].resources`                              | 
+| `Args`                                         | `spec.containers[i].args`                                   |
+| `Env`                                          | `spec.containers[i].env`                                    |
+
+
+## Druid
+
+#### Deploying Druid with Internally Managed MySQL, PostgreSQL, and ZooKeeper
+KubeDB's latest release introduces support for internally managed dependencies, specifically metadata storage (MySQL or PostgreSQL) and ZooKeeper. This enhancement ensures that the KubeDB operator automatically deploys these components unless the user specifies their own external dependencies.
+
+To deploy a Druid cluster that includes an internally managed MySQL database and ZooKeeper instance, use the following YAML configuration:
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Druid
+metadata:
+  name: druid-quickstart
+  namespace: demo
+spec:
+  version: 28.0.1
+  deepStorage:
+    type: s3
+    configSecret:
+      name: deep-storage-config
+  topology:
+    routers:
+      replicas: 1
+```
+
+If you prefer to use PostgreSQL for metadata storage instead of the default MySQL, simply specify it in the `metadataStorage.type` field as shown below:
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Druid
+metadata:
+  name: druid-quickstart
+  namespace: demo
+spec:
+  version: 28.0.1
+  deepStorage:
+    type: s3
+    configSecret:
+      name: deep-storage-config
+  metadataStorage:
+    type: PostgreSQL
+  topology:
+    routers:
+      replicas: 1
+```
+#### Deploying Druid with TLS secure MySQL and PostgreSQL
+Additionally, in this release, KubeDB  has been enhanced to support integration of `Druid` with TLS-secured `MySQL` and `PostgreSQL` clusters.
+
+## Elasticsearch
+
+In this release, In this release, API version for Elasticsearch has been upgraded to `v1`. The new API version is [kubedb.com/v1](http://kubedb.com/v1).
+
+1. In `v1alpha2` API, resources for dedicated topology cluster resources have to be defined at `.spec.topology.<node-type>.resources`, NodeSelector labels have to be defined at `.spec.topology.<node-type>.nodeSelector` and Tolerations have to be defined at `.spec.topology.<node-type>.tolerations.
+Now, in `v1` API version resources has to be defined in `.spec.topology.<node-type>.podTemplate.containers[].resources`. NodeSelector labels have to be defined at `.spec.topology.<node-type>.podTemplate.containers[].nodeSelector` and Tolerations have to be defined at `..spec.topology.<node-type>.podTemplate.containers[].tolerations.
```

Comment:
`.spec.topology.<node-type>.podTemplate.containers[].nodeSelector` -> `.spec.topology.<node-type>.podTemplate.spec.nodeSelector`

`..spec.topology.<node-type>.podTemplate.containers[].tolerations. `->  `.spec.topology.<node-type>.podTemplate.spec.tolerations`.

Reply:
what does that mean? @Neaj-Morshad-101

Reply:
@raihankhan vai can you review this and update if needed?